devlib/utils/ssh.py

```diff
@@ -235,7 +237,11 @@ def background(self, command, stdout=subprocess.PIPE, stderr=subprocess.PIPE, as
 
     def close(self):
         logger.debug('Logging out {}@{}'.format(self.username, self.host))
-        self.conn.logout()
+        try:
+            self.conn.logout()
+        except:
+            logger.warn('Connection lost.')
```

Comment:
`logger.warning` (`warn` was deprecated, IIRC). Also, I would probably change this to `debug` instead --  a warning should be used to make the user aware of something that might require a remedial action. In this case, since we're closing the connection anyway, it doesn't really matter.

Reply:
Updated.